server/api/batches.py

```diff
@@ -0,0 +1,81 @@
+import io, csv
+from flask import jsonify
+
+from . import api
+from ..auth import with_jurisdiction_access
+from ..models import *  # pylint: disable=wildcard-import
+from ..util.csv_download import csv_response, election_timestamp_name
+from ..util.jsonschema import JSONDict
+
+
+@api.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/round/<round_id>/batches/retrieval-list",
+    methods=["GET"],
+)
+@with_jurisdiction_access
```

Comment:
when I haven't reviewed the code in a few days, I keep asking myself "does this call check that it has jurisdiction *and* election access?" I'm wondering if for more casual readers of the code, calling this `@with_jurisdication_and_election_access` might be helpful.

Not required, just musing.

Reply:
yeah, i hear that. i think these would actually be better described according to user type. i would call this `with_jurisdiction_admin_access`